<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>921</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="2" column="2">
      <widget class="QLineEdit" name="Ip_Edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Ip_Lable">
       <property name="text">
        <string>ip:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="Email_Edit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="Pwd_Edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Pwd_Lable">
       <property name="text">
        <string>密码: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Email_Lable">
       <property name="text">
        <string>邮箱: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Mask_Lable">
       <property name="text">
        <string>子网掩码:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="Mask_Edit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1084</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
